### ISSUE TITLE
Cryptic error message when nuxt.config.js is referencing a non-existent plugin

### ISSUE THREAD

**Issue by nyl-auster** [nuxt/nuxt.js#1744]
Got this error and took me some time to figure out that a plugin file was missing in my plugins directory :D 
A message like "file ~/plugins/hydrate-layout-data has not been found" may be more helpful

```
This dependency was not found:

* plugin3 in ./.nuxt/index.js

To install it, you can run: npm install --save plugin3
```

**Comment by vuchl** [comment]
#1198 
#1162 
also show this behavior.

the problem sources here:
https://github.com/nuxt/nuxt.js/blob/53dc5228494e2fbe35e50a864d8af67fb014c7e3/lib/app/index.js#L13
the plugins array is built here:
https://github.com/nuxt/nuxt.js/blob/dcf35933c07ce9b4348d751c80d03c8ca04c2747/lib/builder/builder.js#L66

maybe the `name` could be set to the plugin file name.

Would be a nice issue for https://hacktoberfest.digitalocean.com/ just add a tag hacktoberfest ( @Atinux @alexchopin )

**Comment by himanshujariyal** [comment]
@vuchl I'd like to fix this.

**Comment by jfroffice** [comment]
#1846

**Comment by pi0** [comment]
Fixed in #1846
